go/vt/vtgate/engine/ordered_aggregate_test.go

```diff
@@ -790,3 +790,185 @@ func TestOrderedAggregateExecuteGtid(t *testing.T) {
 	)
 	assert.Equal(t, wantResult, result)
 }
+
```

Comment:
the tests in this file are a little difficult to read. grouping columns and aggregating columns are expected to have very similar results, which makes it hard to overview and understand